diff -urN MyGUI3.2/MyGUIEngine/CMakeLists.txt MyGUI3.2-patched/MyGUIEngine/CMakeLists.txt
--- MyGUI3.2/MyGUIEngine/CMakeLists.txt	2011-12-30 06:56:16.000000000 +1100
+++ MyGUI3.2-patched/MyGUIEngine/CMakeLists.txt	2012-01-07 02:23:11.522759939 +1100
@@ -24,14 +24,6 @@
 		OUTPUT_VARIABLE ${variable})
 ENDMACRO(Subversion_GET_REVISION)
 
-Subversion_GET_REVISION(ProjectRevision)
-if (NOT "${ProjectRevision}" STREQUAL "")
-	add_definitions(-DMYGUI_SVN_REVISION=${ProjectRevision})
-	MESSAGE(STATUS "Using svn revision ${ProjectRevision}.")
-else ()
-	MESSAGE(STATUS "Unknown svn revision. It won't be included into build.")
-endif ()
-
 add_definitions(-DMYGUI_BUILD)
 
 if (MSVC)
