diff -urN MyGUI3.2/Platforms/Ogre/OgrePlatform/CMakeLists.txt MyGUI3.2-patched/Platforms/Ogre/OgrePlatform/CMakeLists.txt
--- MyGUI3.2/Platforms/Ogre/OgrePlatform/CMakeLists.txt	2011-03-21 03:59:50.000000000 +1000
+++ MyGUI3.2-patched/Platforms/Ogre/OgrePlatform/CMakeLists.txt	2012-10-21 02:39:53.579614966 +1100
@@ -12,7 +12,7 @@
 
 add_dependencies(${PROJECTNAME} MyGUIEngine)
 
-target_link_libraries(${PROJECTNAME} ${OGRE_LIBRARIES})
+target_link_libraries(${PROJECTNAME} ${OGRE_LIBRARIES} -ldl -lboost_system)
 link_directories(${OGRE_LIB_DIR})
 
 install(FILES ${HEADER_FILES}
