diff -urN MyGUI_3.2.0/MyGUIEngine/CMakeLists.txt MyGUI_3.2.0-patched/MyGUIEngine/CMakeLists.txt
--- MyGUI_3.2.0/MyGUIEngine/CMakeLists.txt	2012-02-22 22:27:46.000000000 +1100
+++ MyGUI_3.2.0-patched/MyGUIEngine/CMakeLists.txt	2013-06-19 20:51:58.670771079 +1100
@@ -14,26 +14,6 @@
 	add_definitions(-DMYGUI_USE_FREETYPE)
 endif ()
 
-# Get Svn revision
-MACRO(Subversion_GET_REVISION variable)
-	FIND_PROGRAM(SVN_EXECUTABLE svnversion
-		DOC "subversion command line client")
-
-	EXECUTE_PROCESS(
-		COMMAND ${SVN_EXECUTABLE} ${MYGUI_SOURCE_DIR}
-		OUTPUT_STRIP_TRAILING_WHITESPACE
-		OUTPUT_VARIABLE ${variable})
-	#MESSAGE(STATUS  ${MYGUI_SOURCE_DIR} ${variable})
-ENDMACRO(Subversion_GET_REVISION)
-
-Subversion_GET_REVISION(ProjectRevision)
-if (NOT "${ProjectRevision}" STREQUAL "")
-	add_definitions(-DMYGUI_SVN_REVISION="${ProjectRevision}")
-	MESSAGE(STATUS "Using svn revision ${ProjectRevision}.")
-else ()
-	MESSAGE(STATUS "Unknown svn revision. It won't be included into build.")
-endif ()
-
 add_definitions(-DMYGUI_BUILD)
 
 if (MSVC)
